flake.nix

```diff
@@ -0,0 +1,52 @@
+{
+  description = "gpui-component";
+
+  inputs = {
+    nixpkgs.url      = "github:NixOS/nixpkgs/nixos-unstable";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+    flake-utils.url  = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, rust-overlay, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        overlays = [ (import rust-overlay) ];
+        pkgs = import nixpkgs {
+          inherit system overlays;
+        };
+      in
+      {
+        devShells.default = with pkgs; mkShell {
+          buildInputs = [
+            openssl
+            pkg-config
+            eza
+            fd
```

Comment:
I think we can remove `eza`, `fd`. 

And I am not sure is `pango`, `atkmm`, `libsoup_3`, `gdk-pixbuf` is required? 

If that is not needed, we should keep this clean.